docs/guides/migration_mechanism.md

```diff
@@ -9,32 +9,32 @@ keywords:
 - Firefly
 - wallet
 - legacy nodes
+- explanation
 ---
 
 # How the IOTA Chrysalis Phase 2 Token Migration Works
 
-For the transition to Chrysalis Phase 2, the IOTA Foundation had to develop a way for funds to be migrated to the new
-network. The reason a migration mechanism was necessary was because the previous network no longer supported WOTS addresses.
+For the Chrysalis Phase 2 transition, the IOTA Foundation had to develop a way for funds to be migrated to the new network. Because the previous network did not support the WOTS addresses, this new mechanism was created.
 
-The developed mechanism features the following properties:
+The developed mechanism includes that:
```

Comment:
I'm not sure about this phrase and the following bullet points.

Reply:
Agreed